jdbc/src/java/org/apache/hive/jdbc/HttpRequestInterceptorBase.java

```diff
@@ -19,21 +19,33 @@
 package org.apache.hive.jdbc;
 
 import java.io.IOException;
+import java.util.HashMap;
 import java.util.Map;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.function.Supplier;
+import org.apache.hadoop.hive.conf.Constants;
 import org.apache.http.Header;
 import org.apache.http.HttpException;
 import org.apache.http.HttpRequest;
 import org.apache.http.HttpRequestInterceptor;
 import org.apache.http.client.CookieStore;
 import org.apache.http.protocol.HttpContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public abstract class HttpRequestInterceptorBase implements HttpRequestInterceptor {
+  public static final Logger LOG = LoggerFactory.getLogger(HiveConnection.class.getName());
```

Comment:
Can this be made `private`? Also, is it intentional that this class is reusing the `HiveConnection` logger? (Are you looking at `HttpRequestInterceptorBase` as a natural extension of `HiveConnection` from a logging perspective?)

Reply:
yeah, should be private, I'll change

regarding logger: yeah, I admit I haven't investigated why log messages from this class don't make their way to the console, it's mostly because of log4j config, I'm checking it first